kivy/uix/behaviors/compoundselection.py

```diff
@@ -681,6 +687,12 @@ def deselect_node(self, node):
             This method must be called by the derived widget using super if it
             is overwritten.
         '''
+        self._last_selected_node = None
+        self.reslove_last_node()
```

Comment:
Still not right. It's "self._resolve_last_node()". This does not give me much faith that you are running and testing your changes?